allennlp/tests/training/metrics/quoref_em_and_f1_test.py

```diff
@@ -0,0 +1,17 @@
+# pylint: disable=no-self-use,invalid-name
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.training.metrics import QuorefEmAndF1
+
+
+class QuorefEmAndF1Test(AllenNlpTestCase):
```

Comment:
I'd put a comment in here saying that this is intentionally very light on tests, because we are just passing off to the drop metric, which has a lot of tests.

Reply:
I removed this class along with `QuorefEmAndF1`. I added a comment similar to the one you suggested in the tests I have for the evaluation script.